router/pkg/qlog/provider/local.go

```diff
@@ -34,39 +35,23 @@ func (dw *LocalQlog) DumpQuery(ctx context.Context, fname string, q string) erro
 	return nil
 }
 
-func (dw *LocalQlog) Recover(ctx context.Context, path string) ([]string, error) {
+func (dw *LocalQlog) Recover(ctx context.Context, path string) (string, error) {
 	if _, err := os.Stat(path); os.IsNotExist(err) {
 		spqrlog.Logger.Printf(spqrlog.LOG, "%s spqrlog does not exist", path)
-		return []string{}, err
+		return "", err
 	}
 
 	spqrlog.Logger.Printf(spqrlog.LOG, "%s found", path)
-	file, err := os.Open(path)
-	if err != nil {
-		return nil, err
-	}
-	defer func(file *os.File) {
-		err := file.Close()
-		if err != nil {
-			spqrlog.Logger.PrintError(err)
-		}
-	}(file)
-
-	scanner := bufio.NewScanner(file)
-	scanner.Split(bufio.ScanLines)
 
-	var queries []string
-	for scanner.Scan() {
-		line := scanner.Text()
-		query := strings.TrimSpace(line)
-		if len(query) > 0 {
-			queries = append(queries, query)
-		}
+	// TODO it just reads the whole file, but in the future it should read one sql query at a time
+	file, err := os.Open("input.txt")	
```

Comment:
init sql path here?
also there is no sence for doing any cmds on router in local setup
at least in seems to be

Reply:
Yes, I'm agree. Maybe in future we'll remove it to somewhere else.